recipes/scaden/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "scaden" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c5c3c58aa9a9c36151dcbd746445ed038fbc33e7906529a2bbff1d84dcfb7239
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+    - twine
+  run:
+    - python
+    - pandas ==0.21
+    - numpy
+    - scikit-learn
+    - scipy
+    - seaborn
+    - tensorflow ==1.10.0
```

Comment:
Same as above - can this be a range?

Reply:
Yes, it just needs to be below 1.13.0 actually, so I'll change that too.

Reply:
Yes it actually can be a range, just has to be below tensorflow 2. I'll change it accordingly.